src/components/Layouts/_index.scss

```diff
@@ -0,0 +1,104 @@
+////
+/// Modular layouts styles.
+/// @group modular-layouts
+/// @copyright IBM Security 2020
+////
+
+@import '@carbon/colors/scss/colors';
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/theme-maps';
+
+/// Generate a map of spacing tokens from the provided Carbon theme map.
+/// @access private
+/// @param {Map} $theme-map - Map of theme variables.
+/// @returns {Map} - The filtered map of only spacing variables.
+@function get-spacing-tokens-map($theme-map) {
```

Comment:
Because only [Dart Sass supports `@use`](https://sass-lang.com/documentation/at-rules/use), we should practice namespacing any publicly accessible functions, mixins, and variables (as an extra precaution) to avoid potential conflicts with consumers' own Sass builds - A previous example of this was with the `component` mixin, in which our implementation ended up overwriting the consumers and caused errors in their compilation.

Reply:
Yes, I realize this. I made a couple comments throughout asking for guidance on naming. 🙂 I’m hoping for naming suggestions here and elsewhere. Thank you.